### ISSUE TITLE
Create override generator

### ISSUE THREAD

**Issue by aldesantis** [solidusio/solidus_dev_support#102]
It would be useful to have an override generator that creates the right placeholder module in the right path.

**Comment by peterberkenbosch** [comment]
I like this idea, were you thinking about something like this:

```
PBC ~/code/my_solidus_extentionm # ./bin/override Spree::Product MyDecorator
```

- adds override file `my_decorator.rb` with mod..prepend on `Spree::Product`
- adds `my_decorator_spec.rb`
- ?

**Comment by aldesantis** [comment]
@peterberkenbosch correct, that was more or less the idea! Instead of having a `bin/override` though, we can just use the existing CLI.

**Comment by peterberkenbosch** [comment]
Should we still call them decorators? There has been, or is a discussion going on this. we could call it `monkey_patch`? I do not have a very strong opinion on this, but I know others might.

**Comment by aldesantis** [comment]
I like decorators, although recently we have started calling them [prependers](https://github.com/nebulab/prependers). In fact, I think we should hold off on this one for a while because we're still considering whether Prependers should become a standard in the Solidus world.

**Comment by peterberkenbosch** [comment]
I like prependers! Looking at that repo linked again too. Will keep this on hold.
